### PR TITLE
Update commands to not be case sensitive

### LINKED ISSUE #203: [General] Commands should not be case sensitive
Currently using caps for the command will prompt it is invalid, however it still is the correct command.
image

### PR DESCRIPTION
**Pull request by NorbertLoh**
Fixes #203. This fixes both lesson and contacts as only the TutorEase parser was not using `toLowerCase()`.

### CODE REVIEW

**Review by ethantyh692 — APPROVED** [review]
LGTM

File: src/test/java/tutorease/address/testutil/CommandsUtil.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+     */
+    public static String getUpperCaseDeleteContactCommand(Index index) {
+        return ContactCommand.COMMAND_WORD.toUpperCase() + " " + DeleteContactCommand.COMMAND_WORD
+                + " " + index.getOneBased();
+    }
+
+    /**
+     * Returns an uppercase edit command string for editing {@code index}.
+     *
+     * @param descriptor EditPersonDescriptor object.
```
Comment (Ma-Xizi) [inline src/test/java/tutorease/address/testutil/CommandsUtil.java:46]: Should we include all parameters in the JavaDoc?

File: src/test/java/tutorease/address/testutil/PersonUtil.java [hunk 2]
```diff
@@ -44,6 +44,21 @@ public static String getPersonDetails(Person person) {
         return sb.toString();
     }
 
+    /**
+     * Returns the part of command string for the given {@code person}'s details with role.
```
Comment (Ma-Xizi) [inline src/test/java/tutorease/address/testutil/PersonUtil.java:48]: The javaDoc comment should be 

`Returns the part of command string for the given {@code person}'s details without role.`

File: src/test/java/tutorease/address/testutil/CommandsUtil.java (hunk 1 above)
Comment (NorbertLoh) [reply src/test/java/tutorease/address/testutil/CommandsUtil.java:46]: Good catch!

**Pull request merged**